frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -1335,79 +1431,45 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
             const createQueryTab = async (): Promise<void> => {
                 if (searchParams.open_view) {
                     // Open view
-                    const viewId = searchParams.open_view
-
-                    if (values.dataWarehouseSavedQueries.length === 0) {
-                        await dataWarehouseViewsLogic.asyncActions.loadDataWarehouseSavedQueries()
-                    }
-
-                    const view = values.dataWarehouseSavedQueries.find((n) => n.id === viewId)
-                    if (!view) {
-                        lemonToast.error('View not found')
-                        return
+                    const view = dataWarehouseViewsLogic.values.dataWarehouseSavedQueryMapById[searchParams.open_view]
+                    if (view) {
```

Comment:
There's no `else` block to this, meaning the user gets nothing if we can't find the view in the cache - we had a `lemonToast.error(...)` before